mvpa2/tests/test_svm.py

```diff
@@ -187,6 +187,30 @@ def test_C_on_int_dataset(self, clf):
         a[0, 0] = 322           # the value which would overflow
         self.assertTrue(np.isfinite(clf._get_default_c(a)))
 
+
+    def test_memleak(self):
+        skip_if_no_external('libsvm')
+        if __debug__:
+            # to minimize patch in this commit, will not RF to move get_vmem
+            # outside of debug
+            from mvpa2.base.verbosity import get_vmem
+        else:
+            raise SkipTest("for now operates only in __debug__ mode")
+
+        ds = mvpa2.clfs.base.Dataset.from_wizard(
+            samples=np.arange(400000).reshape([4, 100000]),
+            targets=np.arange(4))
+
+        for _ in range(5):
+            svm = mvpa2.clfs.svm.SVM(svm_impl="EPSILON_SVR", C=1)
+            svm.train(ds)
+            svm.predict(ds)
+            gc.collect()
+            if not _:
```

Comment:
I find this hard to read, as the underscore is typically used as a throw-away variable. How about: use ``i`` as loop variable, then
```
is_first_iter = (i==0)
if is_first_iter:
    mem0 = get_vmem()
```